mythril/laser/plugin/plugins/plugin_annotations.py

```diff
@@ -42,18 +48,31 @@ def __copy__(self):
 
     def get_storage_write_cache(self, iteration: int):
```

Comment:
```suggestion
    def get_storage_write_cache(self, iteration: int):
        return self.storage_written.get(iteration, set())
```
We can clean this function up by using something like this